.github/workflows/ci.yml

```
@@ -126,9 +126,10 @@ jobs:
       matrix:
         php-version:
           - "8.0"
-          - "8.1"
-          - "8.2"
-          - "8.3"
+          #- "8.1"
+          #- "8.2"
+          #- "8.3"
```

Temporary to prevent running too much necessary jobs while testing PHP 8.4
```suggestion
          - "8.1"
          - "8.2"
          - "8.3"
```